### PR TITLE
fix(monitor): ignore curator in camunda cloud

### PR DESCRIPTION
**Pull request by ChrisKujawa**
## Description

In the pod restart panel of our general section we want to see whether a pod (especially a broker) has restarted. In our setup we ignore/filter curator, since it is not really useful and expected to restart. In Camunda cloud we use a different naming pattern, which causes that the pod restart panel shows a lot curator restarts it distracts a lot and is not really useful.

Example:

curator-cloud

We just needed to change the query a bit.

It looks like that the grafana instance was updated and adjusted the dashboard to the new version, which caused a lot of not intended changes. Sorry for that. I thought about just cherry picking my change, but I assume the changes are necessary for the newest version.

## Related issues

closes #

## Definition of Done

_Not all items need to be done depending on the issue and the pull request._

Code changes:
* [ ] The changes are backwards compatibility with previous versions
* [ ] If it fixes a bug then PRs are created to [backport](https://github.com/zeebe-io/zeebe/compare/stable/0.24...develop?expand=1&template=backport_template.md&title=[Backport%200.24]) the fix to the last two minor versions. You can trigger a backport by assigning labels (e.g. `backport stable/0.25`) to the PR, in case that fails you need to create backports manually.

Testing:
* [ ] There are unit/integration tests that verify all acceptance criterias of the issue
* [ ] New tests are written to ensure backwards compatibility with further versions
* [ ] The behavior is tested manually
* [ ] The change has been verified by a QA run
* [ ] The impact of the changes is verified by a benchmark 

Documentation: 
* [ ] The documentation is updated (e.g. BPMN reference, configuration, examples, get-started guides, etc.)
* [ ] New content is added to the [release announcement](https://drive.google.com/drive/u/0/folders/1DTIeswnEEq-NggJ25rm2BsDjcCQpDape)

### CODE REVIEW

File: monitor/grafana/zeebe.json [hunk 1]
```diff
@@ -541,10 +571,11 @@
           "steppedLine": false,
           "targets": [
             {
-              "expr": "1 - kube_pod_container_status_ready{pod=~\".*\", pod!~\"curator.*|worker.*|starter.*\", namespace=\"$namespace\"}",
+              "expr": "1 - kube_pod_container_status_ready{pod=~\".*\", pod!~\".*curator.*|worker.*|starter.*\", namespace=\"$namespace\"}",
```
Comment (Zelldon) [inline monitor/grafana/zeebe.json:574]: This is the change

Comment (npepinpe) [reply monitor/grafana/zeebe.json:574]: :1st_place_medal:

File: monitor/grafana/zeebe.json [hunk 2]
```diff
@@ -26,7 +26,7 @@
       "type": "grafana",
       "id": "grafana",
       "name": "Grafana",
-      "version": "6.7.1"
+      "version": "7.0.3"
```
Comment (npepinpe) [inline monitor/grafana/zeebe.json:29]: Hope we don't have any Grafana 6.x deployed anywhere :sweat_smile:

**Review by npepinpe — APPROVED** [review]
:tada:

**Pull request merged**